instrumentation/jaxrs-client/jaxrs-client-1.1/src/main/java/io/opentelemetry/auto/instrumentation/jaxrsclient/v1_1/JaxRsClientV1Instrumentation.java

```diff
@@ -83,45 +81,42 @@ public JaxRsClientV1Instrumentation() {
   public static class HandleAdvice {
 
     @Advice.OnMethodEnter
-    public static SpanWithScope onEnter(
-        @Advice.Argument(0) final ClientRequest request, @Advice.This final ClientHandler thisObj) {
+    public static void onEnter(
+        @Advice.Argument(0) final ClientRequest request,
+        @Advice.This final ClientHandler thisObj,
+        @Advice.Local("otelSpan") Span span,
+        @Advice.Local("otelScope") Scope scope) {
 
       // WARNING: this might be a chain...so we only have to trace the first in the chain.
       boolean isRootClientHandler = null == request.getProperties().get(CONTEXT_ATTRIBUTE);
       if (isRootClientHandler) {
-        Span span =
-            TRACER
-                .spanBuilder(DECORATE.spanNameForRequest(request))
-                .setSpanKind(CLIENT)
-                .startSpan();
-        DECORATE.afterStart(span);
-        DECORATE.onRequest(span, request);
+        span = TRACER.startSpan(request);
 
         Context context = withSpan(span, Context.current());
         request.getProperties().put(CONTEXT_ATTRIBUTE, context);
 
         OpenTelemetry.getPropagators()
             .getHttpTextFormat()
             .inject(context, request.getHeaders(), SETTER);
-        return new SpanWithScope(span, withScopedContext(context));
+        scope = withScopedContext(context);
```

Comment:
it can't.  SETTER is HttpTextFormat.Setter<MultivaluedMap>type.. getSetter() supports HttpTextFormat.Setter<REQUEST>. 
This is the issue i brought up in the very first PR.

Reply:
But you can make `io.opentelemetry.auto.instrumentation.jaxrsclient.v1_1.InjectAdapter implements HttpTextFormat.Setter<ClientRequest>`.